### ISSUE TITLE
Extension icons and flexible spaces disappear when toggling full-screen on videos

### ISSUE THREAD

**Issue by VolatileMark** [rafaelmardojai/firefox-gnome-theme#551]
**Describe the bug**
**Sometimes**, when toggling full-screen on a video, the right most extension icon or flexible space disappears.

**To Reproduce**
1. Open any video player that allows full-screen.
2. Enter and then exit full-screen.
3. The right most extension icon (or flexible space) should have disappeared.

**Expected behavior**
The right most extension icon/flexible space remains where it is.

**Setup information**
 - OS: Fedora 37
 - Installation method: flatpak
 - Firefox version: 109.0
 - Theme version: from master branch

**Additional context**
Entering the "customize toolbar" page restores the missing elements.
This problem was already happening in version 108, but I couldn't track it down. Thought I've seen some similar issues about this, like #540 and #492. I still don't quite understand what causes it, as sometimes toggling full-screen works just fine and no elements from the toolbars disappear.

**Comment by VolatileMark** [comment]
Reinstalling Firefox seems to have fixed the issue. I think I wasn't updating the theme properly, and I was still stuck on an older version...

**Comment by mattblaha** [comment]
I still see this happen pretty often even after resetting firefox and updating the plugin to latest. Also on Fedora 37, 109, flatpak.

**Comment by VolatileMark** [comment]
Yeah, this has happened to me again, but it seems like it's even more random than before. I thought it was just me at this point. It's probably good to reopen this issue.

**Comment by mattblaha** [comment]
Yeah it's not just you. I've considered filing a bug a few times, but like you, can't figure out a pattern and figured out I eventually would and then would file a decent bug.

It's been a thing since 108 at least, I've tried switching from the flatpak to the native package, resetting both and reinstalling the theme, it still seems to happen unpredictably.

**Comment by mattblaha** [comment]
I've updated to include this commit -- will post if I see the issue again.

Thanks!

**Comment by rafaelmardojai** [comment]
Is this still happening? I've never been able to reproduce.

**Comment by mattblaha** [comment]
> Is this still happening? I've never been able to reproduce.

It's definitely much less common but I think it still exists, or at least did pretty recently. I maybe see it once every other month rather than every other day at this point. Could be gone, hard to say. I've definitely seen it at least once in the last few months, but it's rare.
